CauldronMods/Controller/Villains/Menagerie/CardSubClasses/MenagerieCardController.cs

```diff
@@ -9,7 +9,19 @@ public abstract class MenagerieCardController : CardController
     {
         protected MenagerieCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
+            base.Card.UnderLocation.OverrideIsInPlay = AreCardsUnderNonEnclosuresInPlay;
+        }
 
+        private bool? AreCardsUnderNonEnclosuresInPlay
```

Comment:
This won't work (only resolved at card constuction) and likely should be in EnclosureCardController

Reply:
Moving to EnclosureCardController won't work since this is for nonenclosure cards. Put as an add trigger in the menagerieCardController and made sure to call that from all inherited classes, which should resolve correctly